spec/nylas/delta_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+describe Nylas::Delta do
+  it "initialize data" do
+    data = {
+      object: "message",
+      account_id: "acc-id",
+      id: "message-id",
+      type: "message",
+      event: "created",
+      cursor: "999",
+      namespace_id: "namespace-id",
+      date: 1_609_439_400,
+      metadata: { key: :value },
+      object_attributes: { object: :attributes }
+    }
+
+    delta = described_class.new(**data)
+
+    expect(delta.object).to eq("message")
+    expect(delta.account_id).to eq("acc-id")
+    expect(delta.id).to eq("message-id")
+    expect(delta.type).to eq("message")
+    expect(delta.event).to eq("created")
+    expect(delta.cursor).to eq("999")
+    expect(delta.namespace_id).to eq("namespace-id")
+    expect(delta.date).to eq(Time.at(1_609_439_400))
+    expect(delta.metadata).to eq(key: :value)
+    expect(delta.object_attributes).to eq(object: :attributes)
```

Comment:
If you have this more comprehensive test, do you need the other ones also?

Reply:
I thought of covering every attribute we have defined in the class in one test. But May be we can split this into multiples so it reads better.